test/utils/getAbsoluteHooksPath.spec.js

```diff
@@ -0,0 +1,89 @@
+import execa from 'execa'
+// import fs from 'fs'
+import path from 'path'
+
+import getAbsoluteHooksPath from '../../src/utils/getAbsoluteHooksPath'
+import * as stubs from './stubs'
+
+describe('getAbsoluteHooksPath', () => {
```

Comment:
Maybe we can simplify this spec if we refactor the utility!

Reply:
Simpler version of the helper available, I kept the 3 same scenarios though, just simplified a bit the matchers.
Not sure if you want me to rewrite some?